lib/common.sh

```diff
@@ -17,6 +17,11 @@ INSTALLSCRIPT="${OPTDIR}/aznfs_install.sh"
 #
 MOUNTMAP="${OPTDIRDATA}/mountmap"
```

Comment:
Here, I feel there should be a consistency in naming vars: MOUNTMAP, AZ_FILES_MOUNTMAP. Maybe we can have existing one as MOUNTMAPv3 and MOUNTMAPv4. It will look cleaner, or otherwise in comments above them we can mention some more detail how are they different from each other, like one being used for v3 and other v4.

Reply:
I updated the variable naming to MOUNTMAPv3 and MOUNTMAPv4, but I kept the actual file name for NFSv3 to "mountmap". The reason for keeping this file name is to make it simpler to update from older versions of mount helper to this version and after.